recipes/toil/meta.yaml

```
@@ -0,0 +1,126 @@
+# copied from https://github.com/bioconda/bioconda-recipes/blob/9c08a16bac9b7d0a076578823d5008b8bbfc466e/recipes/toil/meta.yaml
+{% set name = "toil" %}
+{% set version = "7.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: c3f4e9567754f5ad6fe56b90913b7c1c9aaade32b545502a483cd57f70f24777
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation --no-cache-dir -vvv
+  entry_points:
+    - toil = toil.utils.toilMain:main
+    - _toil_worker = toil.worker:main
+    - toil-cwl-runner = toil.cwl.cwltoil:main
+    - toil-wdl-runner = toil.wdl.wdltoil:main
+    - toil-wes-cwl-runner = toil.server.cli.wes_cwl_runner:main
+    - _toil_mesos_executor = toil.batchSystems.mesos.executor:main
+    - _toil_contained_executor = toil.batchSystems.contained_executor:executor
+
+requirements:
+  host:
+    - python {{ python_min }}
+    - pip
+    - setuptools >=42
+    - wheel
+  run:
+    - python >={{ python_min }}
+    - dill >=0.3.2,<0.4
+    - requests >2,<=2.31.0
+    - docker-py >=3.7.2,<8
+    - urllib3 >=1.26.0,<3
+    - python-dateutil
+    - psutil >=3.0.1,<6
+    - pypubsub >=4.0.3,<5
+    - addict >=2.2.1,<2.5
+    - pytz >=2012
+    - enlighten >=1.5.2,<2
+    - configargparse >=1.7,<2
+    - typing-extensions >=4.6.2,<5
+    - pyyaml >=6,<7
+    #mesos pymesos isn't available
+    #server reqs
+    # - wes-service is missing
+    # so commenting out the remaining server reqs for now
+    # - connexion >=2.10.0, <3
+    # - flask >=2.0,<3
+    # - werkzeug >=2.0,<3
+    # - flask-cors ==4.0.0
+    # - gunicorn ==21.2.0
+    # - celery >=5.1.0, <6
+    #aws reqs
+    # mypy-boto3-{iam,sdb} are not yet in Conda
+    # - boto >=2.48.0, <3
+    # - boto3 >=1.20.46,<2.0
+    # - boto3-stubs ==1.24.0
+    - botocore
+    - mypy-boto3-s3
+    # - mypy-boto3-sdb ==1.24.0
+    # - mypy-boto3-iam ==1.24.0
+    #encryption reqs
+    - pynacl >=1.4.0
+    #google reqs
+    - apache-libcloud >=2.2.1
+    - google-cloud-storage >=2,<=2.8.0
+    #cwl reqs
+    - cwltool ==3.1.20240112164112
+    - schema-salad >=8.4.20230128170514
+    - galaxy-tool-util <23
+    - ruamel.yaml >=0.15,<=0.18.3
+    - ruamel.yaml.clib >=0.2.6
+    - networkx <4
```

```suggestion
    #cwl reqs
    - cwltool ==3.1.20240508115724
    - schema-salad >=8.4.20230128170514,<9
    - galaxy-tool-util <25
    - ruamel.yaml >=0.15,<=0.19
    - ruamel.yaml.clib >=0.2.6
    - networkx <4
    - cachecontrol-with-filecache
```
Synchronize with https://github.com/DataBiosphere/toil/blob/releases/7.0.0/requirements-cwl.txt